bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/component.jsx

```diff
@@ -11,6 +11,7 @@ import DropdownList from '/imports/ui/components/dropdown/list/component';
 import DropdownListItem from '/imports/ui/components/dropdown/list/item/component';
 import LockViewersContainer from '/imports/ui/components/lock-viewers/container';
 import BreakoutRoom from '/imports/ui/components/actions-bar/create-breakout-room/component';
+import userListService from '/imports/ui/components/user-list/service';
```

Comment:
Should probably import this from the container I think.

Reply:
Sure, I'll correct it.